it/connectors/v2/PenaltyDetailsConnectorISpec.scala

```diff
@@ -31,10 +31,73 @@ class PenaltyDetailsConnectorISpec extends IntegrationSpecCommonBase with ETMPWi
 
   "getPenaltyDetails" should {
     "return a successful response when called" in new Setup {
-      enableFeatureSwitch(CallAPI1812ETMP)
-      mockResponseForNewETMPPayload(Status.OK, "VATC/VRN/123456789")
-      val result: HttpResponse = await(connector.getPenaltyDetails("VATC/VRN/123456789"))
-      result.status shouldBe Status.OK
+      enableFeatureSwitch(CallAPI1811ETMP)
+      mockResponseForGetPenaltyDetails(Status.OK, "VATC/VRN/123456789", body = Some("{}"))
```

Comment:
Could you remove the body parameter so that we can test that the parser works correctly?